retrieval/target.go

```diff
@@ -88,11 +89,8 @@ func (t TargetHealth) value() model.SampleValue {
 }
 
 const (
-	// HealthUnknown is the state of a Target before it is first scraped.
```

Comment:
Right after `// XYZ implements the XYZer interface` – why do we even have implicit interfaces?